packages/mirinae/src/inputs/dropdown/filterable-dropdown/type.ts

```diff
@@ -1,22 +1,19 @@
 import type { MenuItem } from '@/inputs/context-menu/type';
 
-export type FilterableDropdownMenuItem = MenuItem;
+export type DropdownMenuItem = MenuItem;
```

Comment:
question: I think console app imports this... isn't it?

Reply:
I'll check! thx!